core/annotations.h

```diff
@@ -48,7 +48,7 @@
  * @brief Annotation handler registration routines.
  */
 
-#ifdef CLIENT_INTERFACE
+# ifdef CLIENT_INTERFACE
```

Comment:
done, I've used `/* around entire file */` as this is already used in annotations.h